build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts

```diff
@@ -144,9 +143,6 @@ fun DevelocityConfiguration.extractCiData() {
             }
             buildFinished {
                 println("##teamcity[setParameter name='env.GRADLE_RUNNER_FINISHED' value='true']")
-                if (failures.isEmpty() && isRetryBuild) {
-                    println("##teamcity[buildStatus status='SUCCESS' text='Retried build succeeds']")
-                }
             }
```

Comment:
❓ Why change this?

Reply:
`isRetryBuild` causes a CC store problem here, and required changes.
While we were at this we took the opportunity to also improve how this works and thus moved it out to TC.